enocean/README.md

```diff
@@ -342,40 +418,50 @@ EnOcean_Item:
             enocean_rx_key = VALUE
     
     [[actor1B]]
-        enocean_rx_id = FFAABBCD
+        remark = Eltako FSR61, FSR61NP, FSR61G, FSR61LN, FLC61NP - Switch for Ligths
+        enocean_rx_id = 1A794D3
         enocean_rx_eep = F6_02_03
         [[[light]]]
             type = bool
-            enocean_rx_key = B
             enocean_tx_eep = A5_38_08_01
-            enocean_tx_id_offset = 2
+            enocean_tx_id_offset = 1
+            enocean_rx_key = B
+            block_switch = 'False'
+            cache = 'True'
+            enforce_updates = 'True'
+            visu_acl = rw
```

Comment:
These `'True'` and `'False'` values looks strange to me. If you would introduce the `items_attributes` in `plugin.yaml` you could benefit from the metadata handling, which will convert some value to a boolean value if the setting is declared as bool.

Maybe you could start to introduce a `plugin.yaml` - could also obsolete the `remark` setting because this file includes the documentation for the item settings too.

Reply:
Thank you! that was a copy an paste issue.
Corrected.